Source/Core/InputCommon/ControllerEmu/StickGate.h

```diff
@@ -78,15 +83,24 @@ class ReshapableInput : public ControlGroup
 
   virtual ControlState GetGateRadiusAtAngle(double ang) const = 0;
   virtual ReshapeData GetReshapableState(bool adjusted) = 0;
+  virtual ControlState GetDefaultInputRadiusAtAngle(double ang) const;
 
-protected:
-  void AddReshapingSettings(ControlState default_radius, ControlState default_shape,
-                            int max_deadzone);
+  void SetCalibrationToDefault();
+  void SetCalibrationToZero();
+  void SetCalibrationFromGate(const StickGate& gate);
+  void UpdateCalibration(ControlState x, ControlState y);
 
+  void LoadConfig(IniFile::Section* sec, const std::string& defdev,
+                  const std::string& base) override;
+  void SaveConfig(IniFile::Section* sec, const std::string& defdev,
```

Comment:
Same down here regarding readability of shortened variable names. Actually, what the heck is `defdev` supposed to be? Default ...Device?

Reply:
Yeah. I'll make them actual words. :P